bazel/api-gen/rendering/templates/docs-pill-row.tsx

```diff
@@ -15,11 +15,11 @@ export function DocsPillRow(props: {links: LinkEntryRenderable[]}) {
 
   return (
     <nav class="docs-pill-row">
-      {
-        props.links.map((link) => (<a class="docs-pill" href={link.url}>
+      {props.links.map((link) => (
+        <a class="docs-pill" href={link.url} title={link.title}>
```

Comment:
Are we going to run into issues if we have `title=` because `link.title` is undefined?

Reply:
afaik it won't be rendered if its `undefined`.

Reply:
perfect, just wanted to sanity check.